README.md

```diff
@@ -47,36 +67,42 @@ export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 ```
 
-To speedup the transfer of files when kitchens are run on ec2 instances, the [transport](https://docs.chef.io/workstation/config_yml_kitchen/#transport-settings) selected is `kitchen-transport-speedy` https://github.com/criteo/kitchen-transport-speedy.
-
-To install `kitchen-transport-speedy` in the kitchen embedded ruby environment: `/opt/cinc-workstation/embedded/bin/gem install kitchen-transport-speedy`
+To speed up the transfer of files when kitchens are run on ec2 instances,
+the [transport](https://docs.chef.io/workstation/config_yml_kitchen/#transport-settings) selected is [kitchen-transport-speedy](https://github.com/criteo/kitchen-transport-speedy).
+To install `kitchen-transport-speedy` in the kitchen embedded ruby environment please run: `/opt/cinc-workstation/embedded/bin/gem install kitchen-transport-speedy`.
 
-In order to test on docker containers, you also need docker installed on your environment.
+In order to test on docker containers, you also need `docker` installed on your environment.
+Please note that not all the tests can run on docker so in any case we need to validate our recipes on EC2.
```

Comment:
Should we surface the issue we currently have on M1 (different Chef workstation versions required to run ChefSpec and Kitchen tests)? Unless you already fixed it...

Reply:
I'm not aware of this issue, unless you are talking about the "Running kitchen tests on non `amd64` architectures" section that is already described in the README.